### ISSUE TITLE
Focus indicator on tabs in events has insufficient contrast against its background in 'selected' state 

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#866]
The following AI-windows mode(s) have this issue:
- High Contrast White mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![13 1](https://user-images.githubusercontent.com/22068434/88355556-e3ec1d80-cd19-11ea-9c9a-7c471952125c.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
@iamrafan, this is fixed (by adding a contrasting border inside the focus rectangle) in the release found at https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1390.01. Please validate/close at your convenience. Thanks!

**Comment by iamrafan** [comment]
Validated the fix in Canary Release: v1.1.1398.01. All 6 modes are now compliant
